resources/views/listing/partials/item.blade.php

```diff
@@ -10,7 +10,9 @@
                 <img
                     v-if="addToCart.currentThumbnail"
                     :src="'/storage/{{ config('rapidez.store') }}/resizes/200/magento/catalog/product' + addToCart.currentThumbnail + '.webp'"
-                    class="mb-3 h-48 w-full rounded-t object-contain" :alt="item.name" :loading="config.category && count <= 4 ? 'eager' : 'lazy'"
+                    class="mb-3 h-48 w-full rounded-t object-contain"
+                    :alt="item.name"
+                    :loading="config.category && count <= 4 ? 'eager' : 'lazy'"
```

Comment:
If we can get rid of this `config.category` check, we don't need to push it to the frontend in `CategoryController` at all. However, because we're somewhere inside a component we can't directly check `$category`. I chose not to mess with this too much in this PR.